packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -909,29 +910,30 @@ async function determineNodeOptions(
 async function determinePackageBasedOrIntegratedOrStandalone(): Promise<
   'package-based' | 'integrated' | 'standalone'
 > {
+  const pkgMgmt = detectInvokedPackageManager();
+
   const { workspaceType } = await enquirer.prompt<{
     workspaceType: 'standalone' | 'integrated' | 'package-based';
   }>([
     {
       type: 'autocomplete',
       name: 'workspaceType',
-      message: 'npm/yarn monorepo, Nx monorepo, or standalone project?',
+      message: `${pkgMgmt} monorepo, Nx monorepo, or standalone project?`,
       initial: 0,
       choices: [
         {
           name: 'package-based',
-          message:
-            'Package-based Monorepo:     Use a npm/yarn workspace, but have Nx manage the build',
+          message: `Package-based Monorepo:     Creates a workspace that uses ${pkgMgmt} convenstions`,
         },
         {
           name: 'integrated',
           message:
-            'Integrated Monorepo:        Use Nx as the workspace manager and build tools',
+            'Integrated Monorepo:        Creates a workspace that uses path aliases for connecting packages',
```

Comment:
I would refer to them as "tsconfig paths" instead of "path aliases" for extra clarity